cypress/cypress/component/Input/BasicInputField.cy.tsx

```diff
@@ -267,6 +291,29 @@ describe("Input.cy.tsx", () => {
     cy.get("@onSubmitSpy").should("be.calledOnceWith", { [name]: selectedValue });
   });
 
+  it("range input works with step", () => {
+    const name = faker.random.alpha(10);
+    const schema = yup.object().shape({
+      [name]: yup.number(),
+    });
+
+    const [min, selectedValue, max] = faker.helpers.uniqueArray(faker.datatype.number, 3).sort((a, b) => a - b);
+    const step = 2;
+
+    cy.mount(
+      <Form onSubmit={cy.spy().as("onSubmitSpy")} resolver={yupResolver(schema)}>
+        <Input type="range" name={name} label={name} rangeMin={min} rangeMax={max} step={step} />
+
+        <input type={"submit"} />
+      </Form>,
+    );
+
+    cy.get(`input[id=${name}]`).setSliderValue(selectedValue);
+    cy.get(`input[id=${name}]`).should("have.attr", "step", step.toString());
+    cy.get("input[type=submit]").click({ force: true });
+    cy.get("@onSubmitSpy").should("be.calledOnceWith", { [name]: selectedValue });
```

Comment:
@giorgioverzicco What this test is checking differently from the other one already existing for input?

Reply:
@manni497 this one is been removed because is not testable (impossible to manually slide a range)